java/com/aa/act/interview/org/Organization.java

```diff
@@ -19,9 +19,12 @@ public Organization() {
 	 * @param title
 	 * @return the newly filled position or empty if no position has that title
 	 */
-	public Optional<Position> hire(Name person, String title) {
-		//your code here
```

Comment:
if (isValidPosition(title)) {
			Employee newEmployee = new Employee(1, person);
			Position employeePosition = new Position(title, newEmployee);
			Optional<Position> employeeOptional = Optional.of(employeePosition);
			return employeeOptional;
		}